specification/search/resource-manager/Microsoft.Search/stable/2020-03-13/search.json

```diff
@@ -0,0 +1,1964 @@
+{
+  "swagger": "2.0",
```

Comment:
Please incorporate the same comments I made from `2019-10-01-Preview` into here :)
Also the examples :)